pyproject.toml

```diff
@@ -26,7 +26,7 @@ Flask-Cors = "3.0.9"
 apispec = "3.3.0"
 flask-swagger-ui = "3.25.0"
 PyYAML = "5.4"
-python-icat = "0.21.0"
+python-icat = "1.0.0b1"
```

Comment:
It looks like there's a full 1.0.0 release (see https://pypi.org/project/python-icat/#history), could you update the version number please?

Reply:
Updated this to the full release, plus some extra dependencies.